### PR TITLE
ORGS-111: Add docs for the new `createOrganizationsLimit` field

### PR DESCRIPTION
**Pull request by NicolasLopes7**
> [!CAUTION]
> ❌ Blocked by:
> https://github.com/clerk/javascript/pull/3823

> [!IMPORTANT]
> 🔎 Previews:
>
> - https://clerk.com/docs/pr/1364
> - https://clerk.com/docs/pr/1364/organizations/overview
> - https://clerk.com/docs/pr/1364/references/backend/types/backend-user
> - https://clerk.com/docs/pr/1364/references/backend/user/update-user
> - https://clerk.com/docs/pr/1364/references/javascript/user/user

### Explanation:
This PR adds descriptions and indicates the new section of the dashboard to manage create organization limits.

### CODE REVIEW

**Review by royanger — APPROVED** [review]
🚀

File: docs/references/backend/types/backend-user.mdx [hunk 1]
```diff
@@ -39,6 +39,7 @@ The Backend `User` object is similar to the `User` object as it holds informatio
 | `externalAccounts` | [`ExternalAccount[]`][exacc-ref] | An array of all the `ExternalAccount` objects associated with the user via OAuth.<br />**Note**: This includes both verified & unverified external accounts. |
 | `lastActiveAt` | `number` | Date when the user was last active. |
 | `createOrganizationEnabled` | `boolean` | A boolean indicating whether the organization creation is enabled for the user or not. |
+| `createOrganizationsLimit` | `number` | An integer indicating the number of organizations that can be created by the user. If the value is `0`, then the user can create unlimited organizations. |
```
Comment (alexisintech) [inline docs/references/backend/types/backend-user.mdx:42]: Default is 0 ?

File: docs/references/javascript/user/user.mdx [hunk 2]
```diff
@@ -46,6 +46,7 @@ The ClerkJS SDK provides some helper [methods](#methods) on the `User` object to
 | `twoFactorEnabled` | `boolean` | A boolean indicating whether the user has enabled two-factor authentication. |
 | `backupCodeEnabled` | `boolean` | A boolean indicating whether the user has enabled Backup codes. |
 | `createOrganizationEnabled` | `boolean` | A boolean indicating whether the organization creation is enabled for the user or not. |
+| `createOrganizationsLimit` | `number` | An integer indicating the number of organizations that can be created by the user. If the value is `0`, then the user can create unlimited organizations. |
```
Comment (alexisintech) [inline docs/references/javascript/user/user.mdx:49]: Default is 0 ?

**Review by alexisintech — APPROVED** [review]
thank you!!

**Pull request merged**